stateful-datatable-demo/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd" xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.axonivy.market</groupId>
+  <artifactId>stateful-datatable-demo</artifactId>
+  <version>9.4.0-SNAPSHOT</version>
+  <packaging>iar</packaging>
+  <dependencies>
+    <dependency>
+      <groupId>com.axonivy.portal</groupId>
+      <artifactId>portal</artifactId>
```

Comment:
as far as I understood the demo, the only dependency to the Portal is actually the 'PojoConverter' ?
If so; please switch the dependency to be the 'portal-components' ... the full 'portal' should rarely if ever be used as dependency. This should simplify deployments and make this solution easier to maintain.

- [ ] use portal-components rather than portal